doc/tutorial/climada_engine_Impact.ipynb

```diff
@@ -1016,11 +1237,11 @@
       "text/plain": [
-       "<GeoAxesSubplot:title={'center':'Expected annual impact'}>"
+       "<Figure size 900x1300 with 2 Axes>"
       ]
      },
-     "execution_count": 14,
```

Comment:
Something is off with this notebook. The documentation cannot be built because one of the execution counts is missing somewhere. Please do the following:
1. Open a new jupyter notebook instance
2. Restart the kernel
3. Open this file and clear *all* outputs
4. Run all cells from start to finish
5. Save the file
6. Commit the file